lib/elixir/src/elixir_tokenizer.erl

```diff
@@ -343,6 +347,7 @@ tokenize([T | Rest], Line, Column, Scope, Tokens) when T == $); T == $}; T == $]
 
 % ## Two Token Operators
 tokenize([T1, T2 | Rest], Line, Column, Scope, Tokens) when ?two_op(T1, T2) ->
+  maybe_warn_too_many_of_same_char([T1, T2], Rest, Line, Scope),
```

Comment:
This means we will warn on this code: `<<foo>><><<bar>>`. There is no ambiguity here. Maybe we will have to break `two_op` apart.

Reply:
Oh you're right.